Automattic/wp-calypso
# Stats: Adding new empty state wrapper for Top Posts

Related to https://github.com/Automattic/red-team/issues/62

## Proposed Changes

* add new feature flag
* add components for empty states
* render new empty state for Posts and Pages if feature flag is enabled

![SCR-20240625-nmrt](https://github.com/Automattic/wp-calypso/assets/112354940/e79cb48f-4328-4e46-9792-b5fe5e4a0f70)


## Why are these changes being made?


* a part fo the empty states project

## Testing Instructions



* go to the live branch
* apply `stats/empty-module-traffic` feature flag
* verify that pages with empty Top posts components work with and without the feature flag
* repeat for blogs with data

## Pre-merge Checklist
